File: follow-the-beat/src/main/java/com/maco/followthebeat/v2/core/dto/ConcertDTO.java
package com.maco.followthebeat.v2.core.dto;

import lombok.*;

import java.util.UUID;
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class ConcertDTO {
    private UUID id;
    private LocationDTO locationDTO;
    private ArtistDTO artistDTO;
    private ScheduleDTO scheduleDTO;
}

File: follow-the-beat/react-beats/src/types/ConcertDto.ts
import {LocationDto} from "./LocationDto";
import {ArtistDto} from "./ArtistDto";
import {ScheduleDto} from "./ScheduleDto";

export interface ConcertDto {
    id: string;
    locationDTO: LocationDto;
    artistDTO: ArtistDto;
    scheduleDTO: ScheduleDto;
}
